manifests/nsx-operator.yaml

```
@@ -0,0 +1,163 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nsx-ncp-operator
+  namespace: nsx-system-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nsx-ncp-operator
+  namespace: nsx-system-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: nsx-ncp-operator
+  template:
+    metadata:
+      labels:
+        name: nsx-ncp-operator
+    spec:
+      hostNetwork: true
+      serviceAccountName: nsx-ncp-operator
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node.kubernetes.io/not-ready
+      - effect: NoSchedule
+        key: node.kubernetes.io/network-unavailable
```

We should tolerate any taint
```suggestion
      - effect: NoSchedule
```